features/users/blocking.feature

```diff
@@ -119,3 +119,14 @@ Feature: Blocking
       | superadmin       |
       | policy_and_abuse |
       | support          |
+
+  Scenario: Users cannot block more users than the block limit
```

Comment:
I'm wondering if this should be a `spec` instead, since the check happens in the model and not frontend